core/src/main/java/cz/xtf/core/helm/ClusterVersionHelmBinaryPathResolver.java

```diff
@@ -0,0 +1,132 @@
+package cz.xtf.core.helm;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import cz.xtf.core.config.HelmConfig;
+import cz.xtf.core.http.Https;
+import cz.xtf.core.openshift.ClusterVersionInfo;
+import cz.xtf.core.openshift.ClusterVersionInfoFactory;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+class ClusterVersionHelmBinaryPathResolver implements HelmBinaryPathResolver {
```

Comment:
can you please rename this and related tests? Perhaps something like `ConfiguredVersionHelmBinaryPathResolver`